lib/parsers/zip_parser/file_reader.rb

```diff
@@ -18,6 +18,7 @@ def message
       'Could not find the EOCD signature in the buffer - maybe a malformed ZIP file'
     end
   end
+  CDirInfoNotFound = Class.new(Error)
```

Comment:
Do you think we can make the name a little bit more explicit, even if it gets longer? I think it would make exception handling a little bit less cryptic.

Reply:
it makes sense! I changed to `InvalidCentralDirectory`